### DISCUSSION TITLE
Unable to open individual notificiations  in the notifications popup

### DISCUSSION THREAD

**Discussion by andrewebdev** [regolith-linux/regolith-desktop#893] [Q&A]
I really like that the <super + n> view has a list of notifications, but I am unable to click or hit enter on them to open them. Is there something I need to enable for this to work?

**Comment by kgilmer** [comment]
Hmm, what version of Regolith are you running @andrewebdev ?  What kind of keyboard layout do you have?

**Reply by andrewebdev** [reply]
```
   1   │ --- Regolith Diagnostic - Fri 25 Aug 16:14:25 BST 2023
   2   │ --- /etc/regolith/version:
   3   │ REGOLITH_VERSION=2.2
```
I'm using `en-GB` keyboard
